### ISSUE TITLE
Fix accessibility issues

### ISSUE THREAD

**Issue by TGiles** [TGiles/TGiles.github.io#6]
* Anchors need `hover, focus` classes
  * Add underline
  * Change color for better contrast
* Anchors need text
  * Example is `<a href="#about">` which has no text
  * should be a screen reader text fix.
* Other things as they appear

**Comment by TGiles** [comment]
This issue is closed #7
